validate.py

```diff
@@ -157,7 +140,7 @@ def email_results(subject, results_text_string):
     if len(flags) == 0:
         flags.append("GREEN")
     flag_prefix = f"[{', '.join(flags)}] "
-    email_results(flag_prefix + job, results_text)
+    print(results_text)
 
     exit_code = 0
```

Comment:
Per discussion, let's move away from using exit codes for the time being. You can remove this `exit_code` assignment and replace the assignment below with a `logger.error("")` message.